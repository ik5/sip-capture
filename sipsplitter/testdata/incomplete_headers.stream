INVITE foo@bar SIP/2.0
Cont