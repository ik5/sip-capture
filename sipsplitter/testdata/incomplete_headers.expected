{
  "Original": "INVITE foo@bar SIP/2.0\r\nCont",
  "Discarded": 0,
  "NoStartLine": 0,
  "StartLine": 1,
  "NoHeaders": 1,
  "Header": 0,
  "NoBody": 0,
  "Body": 0,
  "Complete": 0,
  "Discards": [],
  "Starts": [
    "INVITE foo@bar SIP/2.0\r\n"
  ],
  "Headers": [],
  "Bodies": [],
  "Messages": []
}